mypy_primer.py

```diff
@@ -1642,6 +1642,10 @@ def inner() -> int | None:
         location="https://github.com/pandas-dev/pandas-stubs",
         mypy_cmd="{mypy} pandas-stubs tests",
     ),
+    Project(
+        location="https://github.com/nion-software/nionutils",
+        mypy_cmd="{mypy} --strict -p nion.utils",
```

Comment:
This will be the default in the next mypy release, but I think you need this for now
```suggestion
        mypy_cmd="{mypy} --namespace-packages --strict -p nion.utils",
```

Reply:
Yes, I dropped that at the last second since this is my lowest level library. I'll resubmit.

Btw, once I work out the 0.981, I'll have several more projects to add, all strictly typed in namespaces.